index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+
+  <body>
+
+    <header id="header">
+    <nav id="nav-bar">
+      <div class="logo"><img src="/images/logo.png" alt="" id="header-img"></div>
+      <div class="menu">
+        <ul>
+          <li><a class="nav-link" href="#services">Features</a></li>
+          <li><a class="nav-link" href="#video">How It Works</a></li>
+          <li><a class="nav-link" href="#price">Pricing</a></li>
+        </ul>
+      </div>
+    </nav>
+    </header>
+
+    <main>
+      <section id="top">
```

Comment:
I would suggest you change the first `section` tag to an `article` tag for the purpose of SEO.